components/modal/style.scss

```diff
@@ -0,0 +1,105 @@
+.edit-post-plugin-modal__editor-modal {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  right: auto;
+  bottom: auto;
+  width: auto;
+  max-width: 90%;
+  max-height: 90%;
+  min-width: 75%;
+  min-height: 75%;
+  border: 0;
+  border-radius: 0;
+  margin-right: -50%;
+  transform: translate(-50%, -50%);
+  background-color: #fff;
+  outline: none;
+  box-shadow: 0 3px 25px #000;
+
+  // In small screens the content needs to be full width.
+  @media ( max-width: #{ ( $break-small ) } ) {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    margin: 0;
+    transform: initial;
+    max-width: none;
+    max-height: none;
+  }
+}
+
+.edit-post-plugin-modal__editor-modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.6);
+  z-index: 999999;
+  // on mobile the main content area has to scroll
+  // otherwise you can invoke the overscroll bounce on the non-scrolling container, causing (ノಠ益ಠ)ノ彡┻━┻
+  @include break-small {
+    position: fixed;
+    top: $admin-bar-height-big;
+  }
+
+  @include break-medium() {
+    top: $admin-bar-height;
+  }
+}
+
+@include editor-left('.edit-post-plugin-screen-takeover__editor-screen-takeover-overlay');
+
+// We prevent scrolling of the body when the React Modal screen takeover is opened.
+body.ReactModal__Body--open {
```

Comment:
Use `bodyOpenClassName` props to overwrite `ReactModal__Body--open`.
While we're on the topic it would be favourable if `portalClassName` was also changed to something that is more suitable to WordPress.

Reply:
renamed to WordPress-modal for now.